cdisc_rules_engine/models/operation_params.py

```diff
@@ -0,0 +1,26 @@
+from dataclasses import dataclass
+from typing import List
+
+import pandas as pd
+
+
+@dataclass
+class OperationParams:
```

Comment:
I like this. As a rule we should probably create dataclasses for all complex function parameters